sdk/elixir/dagger.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "elixir-sdk",
+  "sdk": "go",
+  "include": ["LICENSE", "**/*.ex", "**/*.exs", "**/mix.exs", "**/mix.lock"],
+  "exclude": [
+    "**/.changes",
+    "**/.changie.yaml",
+    "**/README.md",
+    "**/CHANGELOG.md",
+    "**/.credo.exs",
+    "**/test",
+    "**/_build",
+    "**/deps",
+    "**/doc",
+    "**/.elixir-tools",
+    "**/.elixir_ls",
+    "**/.lexical",
+    "**/credo.exs",
+    "**/erl_crash.dump"
+  ],
```

Comment:
Are all of these excludes really necessary? As I said before, it's better to rely more on _include_ and only add to _exclude_ when necessary. The reason is that it's quite easy to add new files to `sdk/elixir` and you don't want to have to remember to add it to the list of excludes.

See if you can **avoid starting the patterns with `**/`** as I think that's more difficult to manage.

One example is if you include 'lib/**/*.ex`, then you don't need to exclude `**/test`, etc.

Reply:
I changed to include only necessary files. The exclude here is not necessary anymore. 🎉